config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -364,36 +364,6 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: containerd-node-e2e-features-1.6
-- interval: 12h
-  name: ci-containerd-soak-cos-gce
```

Comment:
I believe this is the only soak test we have. I fixed it before: https://github.com/kubernetes/kubernetes/issues/107802

Can we keep it and look deeper into the reasons it fails? Perhaps move it to containerd tab - it will be good to highlight that this is not COS specific, it is about soaking

Reply:
Done, moved to `sig-node-containerd` tab.